Runtime/Managers/EditMediator.cs

```diff
@@ -182,13 +156,16 @@ public void Notify<T>(ReupEvent eventName, T payload)
         public async Task ReceiveWebMessage(string serializedWebMessage)
         {
             JObject message = JObject.Parse(serializedWebMessage);
-            if (!incomingMessageValidator.ValidateMessage(serializedWebMessage))
+            IList<string> validationErrors;
+            bool isMessageValid = message.IsValid(RomuloExternalSchema.incomingMessageSchema, out validationErrors);
+            if (!isMessageValid)
             {
+                foreach (string validationError in validationErrors)
+                {
+                    Debug.Log(validationError);
```

Comment:
although it would be way better if we send this array of strings in the error message instead of just logging them to the console. 
```suggestion
                    Debug.LogWarning(validationError);
```